### PR TITLE
Hotfix: openurl in_doaj

### PR DESCRIPTION
**Pull request by Steven-Eardley**
**MERGE OFFLINE**
This prevents OpenURL for asking for TOCs that won't render because their target has ```in_doaj``` set to ```false```. For Issue #694.

N.B. this PR should work properly once #696 has been resolved.

### CODE REVIEW

**Comment by emanuil-tolev** [comment]
> N.B. this PR should work properly once #696 has been resolved.

ok .. so, should this be merged / deployed then? Should we wait till 696 is resolved?

File: portality/models/openurl.py [hunk 1]
```diff
@@ -29,7 +29,9 @@
     'isbn' : ('index.issn.exact', 'index.issn.exact'),
 }
 
-TERMS_SEARCH = { "query" : {"bool" : { "must" : [] } } }
+# Terms search template. Ensure all queries from OpenURL return publicly visible results with in_doaj : true
+IN_DOAJ_TERM = { "term" : { "admin.in_doaj" : True } }
+TERMS_SEARCH = { "query" : {"bool" : { "must" : [ IN_DOAJ_TERM ] } } }
```
Comment (emanuil-tolev) [inline portality/models/openurl.py]: @richard-jones I've noticed you do filtered queries, should we do this or is this OK?

**Comment by Steven-Eardley** [comment]
We don't need to wait - this will make an improvement ahead of a fix for #696

File: portality/models/openurl.py (hunk 1 above)
Comment (emanuil-tolev) [inline portality/models/openurl.py]: ok, right, we can change this later, it'll work like this so merging

**Comment by emanuil-tolev** [comment]
@Steven-Eardley rolled out!

**Pull request closed without merging**